tests/linux_kernel/helpers/test_stack.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from contextlib import redirect_stdout
+import io
+
+from drgn.helpers.common import print_annotated_stack
+from tests.linux_kernel import LinuxKernelTestCase, fork_and_sigwait
+
+
+class TestPrintAnnotatedStack(LinuxKernelTestCase):
+    def test_print_annotated_stack(self):
+        with fork_and_sigwait() as pid:
+            trace = self.prog.stack_trace(pid)
+
+            f = io.StringIO()
+            with redirect_stdout(f):
+                print_annotated_stack(trace)
+
+            printed_trace = f.getvalue()
+
+            assert "slab object: task_struct" in printed_trace
```

Comment:
Use `self.assertIn` for all of these checks so that we get a nicer message if this fails:

```suggestion
            self.assertIn("slab object: task_struct", printed_trace)
```

Reply:
Fixed!